webapp/appModules/home/dashboard/surveyInfo/surveyInfo.js

```diff
@@ -131,13 +131,13 @@ const mapStateToProps = state => {
   const user = AppState.getUser(state)
   const surveyInfo = SurveyState.getStateSurveyInfo(state)
   const lang = Survey.getDefaultLanguage(surveyInfo)
-  const showCollectImportReportButton = false //SurveyState.hasCollectImportReport(state)
+  const isFromCollect = Survey.isFromCollect(surveyInfo)
 
   return {
     surveyInfo,
     lang,
     canEditDef: AuthManager.canEditSurvey(user, surveyInfo),
-    showCollectImportReportButton
+    isFromCollect
```

Comment:
maybe it's useless to add this to the props. since surveyInfo is already a prop,  you could call Survey.isFromCollect(surveyInfo) directly in the render function